README.md

```diff
@@ -117,6 +117,22 @@ jobs:
           name: kubescape-scan-report
           path: results.xml
 ```
+- Exceptions
+
+```yaml
+name: KubeScape-Exceptions
+on: push
+
+jobs:
+  kubescape:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: action/checkout@master
+      - uses: orrgato/Kubescape-github-action@add-exception
```

Comment:
Use `kubescape/github-action@main` for the example

Reply:
Updating it right now.

Reply:
Done. Solves #6 too.